dev-python/nbconvert/nbconvert-9999.ebuild

```diff
@@ -22,9 +22,10 @@ fi
 
 LICENSE="BSD"
 SLOT="0"
-IUSE="test"
+IUSE="doc test"
 
 RDEPEND="
+	>=app-text/pandoc-1.12.1
```

Comment:
Finally found the docs: https://nbconvert.readthedocs.org/en/latest/install.html#pandoc

Anyway, I would appreciate an optional dependency or a hint like previously in ipython (were I will remove it for 4.0.0). Conversion between python, html and markdown is usable without pandoc.